jbmc/regression/jbmc-concurrency/synchronized-blocks/test1.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 A.class
---function 'A.me0:()V' --lazy-methods --cp `../../../../scripts/format_classpath.sh . ../../../src/java_bytecode/library/core-models.jar` --show-goto-functions
+--function 'A.me0:()V' --lazy-methods --cp `../../../../scripts/format_classpath.sh . ../../../src/java_bytecode/library/core-models.jar` --show-goto-functions --java-threading
```

Comment:
I think this test doesn't check that line 26 is actually reached. Maybe add a check for the coverage goal (also in the other .desc files).

Reply:
This test is not testing for coverage, see test4.desc.